collect_app/src/androidTest/java/org/odk/collect/android/support/pages/FormEndPage.java

```diff
@@ -44,6 +44,11 @@ public MainMenuPage clickFinalize() {
         return clickFinalize(new MainMenuPage());
     }
 
+    public MainMenuPage clickSend() {
+        onView(withText(getTranslatedString(R.string.send))).perform(click());
```

Comment:
Yes but:
1. Then I would need to check if we are moved to the Main menu separately
2. Probably I should remove other methods like `clickFinalize`/`clickSaveAsDraft` too
3. `FormEndPage` is small enough so I think it can have special methods like that for those two buttons to make tests clearer.

Reply:
Oh I totally agree about having action specific methods like `clickSend`! I just meant you could replace the line `onView(withText(getTranslatedString(R.string.send))).perform(click());` with `clickOnString(R.string.send)`.

Reply:
Ah I see. Done.